tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/wwwroot/app/directives/common.directive.js

```diff
@@ -512,6 +512,153 @@
             }
         }
     }
+    'use strict';
+    function matteradditionalfieldsdirective($compile) {
+

```

Comment:
Purpose: new directive to render the HTML for the extra fields on Create Matter page.